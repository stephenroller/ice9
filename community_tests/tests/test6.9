# a proc with no statements		fail

proc 
foo 
(
) 
end