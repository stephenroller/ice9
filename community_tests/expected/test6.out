Loading file: generated/test6.tm
-1 
Number of instructions executed = 16
